compiler/frontend/beam/src/main/java/edu/snu/onyx/compiler/frontend/beam/OnyxPipelineVisitor.java

```diff
@@ -246,7 +246,8 @@ private static BeamCoder getCoderForView(final ViewFn viewFn, final Coder beamIn
                                                                                     final IRVertex dst) {
     if (dst instanceof OperatorVertex && ((OperatorVertex) dst).getTransform() instanceof GroupByKeyTransform) {
       return DataCommunicationPatternProperty.Value.Shuffle;
-    } else if (dst instanceof OperatorVertex && ((OperatorVertex) dst).getTransform() instanceof CreateViewTransform) {
+    } else if (dst instanceof OperatorVertex && ((OperatorVertex) dst).getTransform() instanceof CreateViewTransform
```

Comment:
Would it make more sense if this part returned something like a `many-to-one` edge, and get rid of side inputs?

Reply:
By that I mean the DAG should look like 
->(many to one) [CreateViewTrasnform] ->(broadcast)
instead of
-> (broadcast) [CreateViewTransform] -> (broadcast + sideInput)

Reply:
Thanks for the suggestion! 
However, it is not that easy to remove side input, because side input is treated specially in runtime (as you know.)
Of course we can treat all broadcast edges as side input, but adding any extra meaning to the concept of data communication pattern might be dangerous.
Therefore, if you want to remove the side input, we should remove the extra-care for the side input before. It can be handled in another issue.